remixer_storage/src/main/java/com/google/android/libraries/remixer/storage/json/StoredVariable.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.libraries.remixer.storage.json;
+
+import com.google.android.libraries.remixer.RangeVariable;
+import com.google.android.libraries.remixer.RemixerItem;
+import java.util.List;
+
+/**
+ * This is an intermediate format that conforms to the agreed upon cross-platform storage and sync
+ * protocol.
+ *
+ * <p>This is never used for anything other than storage and syncing and is meant to be converted to
+ * a regular {@link com.google.android.libraries.remixer.RemixerItem} as soon as it's completely
+ * parsed.
```

Comment:
I really like the javadoc here. Does this class have similar parallels with SerializableRemixerContents? May want to add some of this wording to that javadoc as well.